tests/unit/components/gallery/HDGalleryMedia.spec.js

```diff
@@ -8,10 +8,26 @@ const wrapperBuilder = wrapperFactoryBuilder(HdGalleryMedia, {
   },
 });
 
+const emptyWrapperBuilder = wrapperFactoryBuilder(HdGalleryMedia, {});
+
 describe('HdGalleryMedia', () => {
   it('renders component', () => {
     const wrapper = wrapperBuilder();
 
     expect(wrapper.html()).toMatchSnapshot();
   });
+
+  it('renders empty element', () => {
+    const wrapper = emptyWrapperBuilder();
+
+    expect(wrapper.html()).toMatchSnapshot();
+  });
```

Comment:
I would challenge it as well, maybe we can tackle it differently.

Does it make sense to have this component without an item? Shall we go in the direction of having it with required props?

Reply:
Is there to cover 
`default: () => ({}),`

Reply:
It does not make much sense is just a little more resilient. 

That test is there to cover HdGalleryMedia:38

Reply:
Yep, I get it that. My only question is if it makes sense. If does, I'm fine but I just don't see it being used like that.

The only place where `HdGalleryMedia` is used is inside `HdGallery` which has a handler for [empty scenario](https://github.com/homeday-de/homeday-blocks/blob/develop/src/components/gallery/HdGallery.vue#L9).

Does it make sense to keep it with a default of `{}`?
IMHO I think the property should be required but I don't know if we have an use case for empty state on this component.

What I'm challenging here is not the implementation itself, it the reasoning behind it.

Reply:
I do agree with you. I'll give it a refactor time